src/editor/koLanguageCommandHandler.p.py

```diff
@@ -2187,3 +2191,13 @@ def getLine(scin, lineNo):
     lineEnd = scin.getLineEndPosition(lineNo)
     line = scin.getTextRange(lineStart, lineEnd)
     return line
+
+def getReflowLine(scin, lineNo, view):
+    import reflow
+    line = reflow.Line(getLine(scin, lineNo));
+    # Check if this is in a block comment
+    lineStart = scin.positionFromLine(lineNo)
+    style = scin.getStyleAt(lineStart)
+    if style in view.languageObj.getCommentStyles() and style == SCE_UDL_SSL_COMMENTBLOCK:
```

Comment:
I'm guessing you did this because the `view` and `SCE_UDL_SSL_COMMENTBLOCK` aren't available in *reflow.py*, but ideally this would happen there.  Could you do a quick investigation to see if you can move this logic into *reflow.py*?  Note that it should be language agnostic.

Reply:
I was running into a few issues.  The big one was I couldn't pass extra parameters to the `Line` constructor because `unicode` has optional arguments.  I refactored, but did that in a separate branch (https://github.com/ssigwart/KomodoEdit/compare/reflowPHPDoc2) in case it's not what you want.  The specific commit is https://github.com/Komodo/KomodoEdit/commit/4f53d8d0c5e93ba8703bab09b2f615744605ffab.  If this is what you are looking for, I can cherry pick it.  If not, let me know.

Reply:
I see what you mean.  That looks more disjointed and the only way I can think of to consolidate all the logic into reflow is to rewrite reflow to do a better job of containing the info it needs.  I don't think that's worth the effort at this point.  This is already great.

Reply:
I couldn't get it out of my head and haven't written code all day so I changed it.

I've moved `getReflowLine` to reflow.py.  I removed `scin` from the args.  The call looks like this `reflow.getLine(lineNo, view)`.  I also cp'd `getLine` into *reflow.py* and the block comment scintilla import statement at the top of the file.

Seems to be doing the same stuff you're original code did.  I'm gonna merge then cherrypick my changes on top.